docs/spelling-wordlist.txt

```diff
@@ -25,3 +25,10 @@ subcommands
 # this "word" (tui) appears nowhere but oh god as usual who wants to debug Sphinx
 tui
 validator
+clojure
```

Comment:
All of these should go away if you don't hardcode the language names presumably.

Reply:
Yes! removing this